jest.setup.ts

```diff
@@ -1,3 +1,10 @@
+// avoid "This script should only be loaded in a browser extension." issue
+// https://github.com/clarkbw/jest-webextension-mock/issues/149#issuecomment-1116307310
+// @ts-ignore: chrome available during tests via jest-webextension-mock
+if (!chrome.runtime) chrome.runtime = {};
```

Comment:
Not actually sure were `chrome` is coming from during the tests. I assume via `web-extension-mock` but not sure how to satisfy TS here.